helm/prometheus-rules/templates/alerting-rules/dex.rules.yml

```diff
@@ -0,0 +1,24 @@
+apiVersion: monitoring.coreos.com/v1
+kind: PrometheusRule
+metadata:
+  creationTimestamp: null
+  labels:
+    {{- include "labels.common" . | nindent 4 }}
+  name: dex.rules
+  namespace: {{ .Values.namespace  }}
```

Comment:
Do you want this alert also for dex instances installed by customers on their workload clusters ?

Reply:
I thought so, since it's a managed app. However, now that I think about it it does not make much sense to get paged when they misconfigure it or play around. Good point.